src/entities/work-log-record.ts

```diff
@@ -0,0 +1,36 @@
+import {Column, Entity, PrimaryGeneratedColumn} from "typeorm";
+import {MaxLength, Property, Required} from "@tsed/common";
+
+@Entity("work_log_record")
+export class WorkLogRecord {
+
+  @PrimaryGeneratedColumn()
+  @Property()
+  id: number;
+
+  @Column({name: "project_id"})
+  @Property()
+  @Required()
+  projectId: string;
+
+  @Column()
+  @MaxLength(200)
+  @Property()
+  @Required()
+  text: string;
+
+  @Column({name: "spent_time"})
+  @Property()
+  @Required()
+  spentTime: number;
```

Comment:
Are we tracking time in seconds or milliseconds? I would recommend to specify that in the column/field name, e.g. `spentTimeMS` or `spentTimeSeconds`